README.md

```diff
@@ -63,25 +63,29 @@ If you find a typo or you feel like you can improve the HTML, CSS, or JavaScript
 [terraform-plugin-framework]: https://github.com/hashicorp/terraform-plugin-framework
 
 
-## Running the Site Locally
+## Previewing Changes
 
-### Using Docker
+You should preview your changes locally to ensure that the content is rendering properly before you create a pull request. The build includes content from this repository and the [`terraform-website`](https://github.com/hashicorp/terraform-website/) repository, allowing you to preview the entire Terraform documentation site.
 
-If you wish to run the site in a container, you can run the site locally via
-`make`.
+To preview your content, complete the following steps:
 
-- `make website`
-  - This command will pull and run the latest website container.
-  - This includes live reload which will load your changes as you make them.
-- `make website/local`
-  - This command will run the website locally using a locally built image
-  - This includes live reload which will load your changes as you make them.
-- `make website/build-local`
-  - This command will build a local image of the website from `hashicorp/dev-portal.git`.
+**Set Up Local Environment**
+
+1. [Install Docker](https://docs.docker.com/get-docker/).
+1. Restart your terminal or command line session.
+
+**Launch Site Locally**
+
+1. Navigate into your local `terraform-plugin-mux` top-level directory and run `make website`. 
```

Comment:
```suggestion
1. Navigate into your local `terraform-docs-common` top-level directory and run `make website`. 
```

Reply:
oh wait!! this isn't even common - it's this directory - thank you!